dotnet/src/IntegrationTests/Connectors/Memory/Postgres/PostgresVectorStoreFixture.cs

```diff
@@ -0,0 +1,263 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Docker.DotNet;
+using Docker.DotNet.Models;
+using Microsoft.Extensions.VectorData;
+using Microsoft.SemanticKernel;
+using Npgsql;
+using Xunit;
+
+namespace SemanticKernel.IntegrationTests.Connectors.Memory.Postgres;
+
+public class PostgresVectorStoreFixture : IAsyncLifetime
+{
+    /// <summary>The docker client we are using to create a postgres container with.</summary>
+    private readonly DockerClient _client;
+
+    /// <summary>The id of the postgres container that we are testing with.</summary>
+    private string? _containerId = null;
+
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="PostgresVectorStoreFixture"/> class.
+    /// </summary>
+    public PostgresVectorStoreFixture()
+    {
+        using var dockerClientConfiguration = new DockerClientConfiguration();
+        this._client = dockerClientConfiguration.CreateClient();
+        this.HotelVectorStoreRecordDefinition = new VectorStoreRecordDefinition
+        {
+            Properties = new List<VectorStoreRecordProperty>
+            {
+                new VectorStoreRecordKeyProperty("HotelId", typeof(ulong)),
```

Comment:
Are we using this anywhere?  From the constants list I don't believe we support ulong, so I would expect using this to fail.